rbac/management/principal/it_service.py

```diff
@@ -275,7 +280,36 @@ def get_service_accounts_group(self, group: Group, bearer_token: str, options: d
             service_account_principals=sap_dict, it_service_accounts=it_service_accounts, options=options
         )
 
-        return service_accounts
+        # If either the description or name filters were specified, we need to only keep the service accounts that
```

Comment:
this filtering is for `GET groups/uuid/principals/` endpoint ... is it expected the same for `GET principals/?`

Reply:
yes I can see now in the ticket that the answer is NO, so for the record this filtering is expected only for groups endpoint